packages/discord-bot/src/handlers/activate.ts

```diff
@@ -24,21 +24,32 @@ export const activationHandler = async (
     await interaction.reply('Unable to create user account.');
     return;
   }
-  const baseURL = process.env.FRONTEND_URL;
-  if (!baseURL || baseURL === undefined) {
+  const server = process.env.SERVER;
+  const port = process.env.FRONTEND_DEVELOPMENT_PORT;
+
+  if (!server || server === undefined) {
+    await interaction.reply(
+      'ERROR: `SERVER` not defined in environment variables. Contact praise admin'
+    );
+    return;
+  }
+  if (!port || port === undefined) {
     await interaction.reply(
-      'ERROR: `FRONTEND_URL` not defined in environment variables. Contact praise admin'
+      'ERROR: `FRONTEND_DEVELOPMENT_PORT` not defined in environment variables. Contact praise admin'
     );
     return;
   }
+  const frontendUrl = `http://${
+    process.env.NODE_ENV === 'production' ? server : `${server}:${port}`
+  }`;
```

Comment:
This conditional should not be necessary. We can change the env variable FRONTEND_URL to include a port if desired.

Reply:
Currently we don't need `FRONTEND_URL` at all. The following variables allow for full customisation: 

```
SERVER
API_PORT
REACT_APP_API_URL
FRONTEND_DEVELOPMENT_PORT
```

The likelihood of us wanting to serve the production frontend on anything else than standard port 80/443 is veery small. Or am I missing something?

Reply:
> Currently we don't need `FRONTEND_URL` at all. The following variables allow for full customisation:
> 
> ```
> SERVER
> API_PORT
> REACT_APP_API_URL
> FRONTEND_DEVELOPMENT_PORT
> ```
> 
> The likelihood of us wanting to serve the production frontend on anything else than standard port 80/443 is veery small. Or am I missing something?

It looks like `FRONTEND_DEVELOMENT_PORT` is only used in this one place -- to define the `frontendUrl` value. So why not just specify the frontend url directly with an env variable?

Then the protocol, host, and domain can all be defined within that variable as needed -- allowing for maximum flexibility to deploy the frontend anywhere without needing to modify the codebase.